interfaces/api/v1/world/bible.py

```
@@ -906,28 +906,28 @@ async def _sse_bible_generator(
+                    bible_generator.bible_service.add_location(
+                        novel_id=novel_id,
+                        location_id=pd["location_id"],
+                        name=pd["name"],
+                        description=pd["description"],
+                        location_type=pd["location_type"],
+                        connections=pd.get("connections", []),
+                        parent_id=pd.get("parent_id"),
+                    )
+                    location_ids.append((pd["location_id"], pd))
+                except Exception:
+                    pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)